librz/util/float/float.c

```diff
@@ -20,6 +20,33 @@
 #include <rz_userconf.h>
 #include <math.h>
 
+/**
+ * define gen_nan and gen_inf for multiple types
+ */
+#define define_types_gen_nan(fname, ftype) \
+	RZ_API inline ftype types_gen_##fname##_nan() { \
+		static ftype zero = 0; \
+		ftype ret = zero / zero; \
+		feclearexcept(FE_ALL_EXCEPT); \
+		return ret; \
+	}
+
+#define define_types_gen_inf(fname, ftype) \
+	RZ_API inline ftype types_gen_##fname##_inf() { \
```

Comment:
there is no real need to set the `inline` param since they are not in a header file.

Reply:
It's possible to force inlining using both `extern` and `inline` though: https://gcc.gnu.org/onlinedocs/gcc/Inline.html
But it's very GCC-specific and likely will not work on other compilers.

Reply:
then I will just remove it.